kamaki/clients/compute/__init__.py

```diff
@@ -441,3 +441,56 @@ def delete_floating_ip(self, tenant_id, fip_id=None):
         """
         r = self.floating_ips_delete(tenant_id, fip_id)
         return r.headers
+
+    def list_keypairs(self):
+        """
+        :returns: (list) [
+            {id: ..., name: ..., public_key: ..., fingerprint: ...},
+            ... ]
+        """
+        r = self.keypairs_get()
+        return r.json['keypairs']
+
+    def get_keypair_details(self, key_name=None):
```

Comment:
What are the semantics of this call: get_keypair_details()
(without parameters).
It seems to me "key_name" must not be optional.